src/Workspaces/CSharp/Portable/Formatting/CSharpFormattingOptions.Parsers.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.CSharp.Formatting
+{
+    public static partial class CSharpFormattingOptions
+    {
+        internal static bool DetermineIfSpaceOptionIsSet(string value, SpacingWithinParenthesesOption parenthesesSpacingOption)
+            => (from v in value.Split(',').Select(v => v.Trim())
+                let option = ConvertToSpacingOption(v)
+                where option.HasValue && option.Value == parenthesesSpacingOption
+                select option)
+                .Any();
+
+        private static SpacingWithinParenthesesOption? ConvertToSpacingOption(string value)
+        {
+            switch (value)
+            {
+                case "expressions": return SpacingWithinParenthesesOption.Expressions;
+                case "type_casts": return SpacingWithinParenthesesOption.TypeCasts;
+                case "control_flow_statements": return SpacingWithinParenthesesOption.ControlFlowStatements;
+                default: return null;
+            }
+        }
+
+        internal static object ParseEditorConfigSpacingAroundBinaryOperator(string binaryOperatorSpacingValue)
+        {
+            switch (binaryOperatorSpacingValue)
+            {
+                case "ignore": return BinaryOperatorSpacingOptions.Ignore;
+                case "none": return BinaryOperatorSpacingOptions.Remove;
+                case "before_and_after": return BinaryOperatorSpacingOptions.Single;
+                default: return BinaryOperatorSpacingOptions.Single;
+            }
+        }
+
+        internal static object ParseEditorConfigLablePositioning(string lableIndentationValue)
+        {
+            switch (lableIndentationValue)
+            {
+                case "left_most": return LabelPositionOptions.LeftMost;
```

Comment:
Probably, but once again these are public type names that are going to need a compat council review to change.